kubectl-plugin/uninstall/component/servicemesh.go

```diff
@@ -0,0 +1,31 @@
+package component
+
+import "github.com/kubesphere-sigs/ks/kubectl-plugin/common"
+
+// ServiceMesh return the struct of ServiceMesh
+type ServiceMesh struct {
+}
+
+// Uninstall uninstall ServiceMesh
+func (s *ServiceMesh) Uninstall() error {
+	_ = common.ExecCommand("curl", "-L", "https://istio.io/downloadIstio")
+	_ = common.ExecCommand("sh", "downloadIstioCandidate.sh")
```

Comment:
We have an easier way to install dependency tools. See also [here](https://github.com/kubesphere-sigs/ks/blob/8f47e9aede950f77932c259b396574291898262d/kubectl-plugin/install/k3d.go#L90)

Reply:
Good idea, istio seems do not support installed by this method, [Istio download](https://istio.io/latest/zh/docs/setup/getting-started/#download). if you a good idea, please help me to improve, thanks a lot.

Reply:
I used `wget` to download `istioctl` binary package, please see 9667b8a